### ISSUE TITLE
Disabling Tideways gives an XHGUI button that leads to a 404

### ISSUE THREAD

**Issue by tomjn** [Varying-Vagrant-Vagrants/VVV#2289]
Currently if the tideways utility is active `xhgui.vvv.test` is added to the hosts file.

If you then disable the tideways utility, this host is not added, but, an XHGUI button is present in the dashboard. This results in a DNS lookup error page

**Comment by Mte90** [comment]
We check if the folder exists like also for the other utils https://github.com/Varying-Vagrant-Vagrants/dashboard/blob/master/php/blocks/sidebar/bundled-tools.php#L27

I guess that removing a utility don't remove the folders for all the bundled tools so we need a better detection

**Comment by tomjn** [comment]
That's not the problem, the problem is that the host for `xhgui.vvv.test` is never added. If it were then XHGUI would be accessible.

Remember, the only way to truly remove a utility is either manually, or `vagrant destroy`. VVV does not remove or uninstall utilities

**Comment by Mte90** [comment]
So where is the issue? we cannot add the host if the utility is not present and at same time is enough that the dashboard doesn't print that button.

**Comment by tomjn** [comment]
Either always add the host, or update the dashboard detection not to show XHGUI. We should not have buttons on our dashboard that lead to broken URLs

**Comment by Mte90** [comment]
Yes so I think that we need a better way to detect if an utility is available in the dashboard that doesn't use the folder way as we are doing now.

**Comment by tomjn** [comment]
Is there any harm in having XHGUI available anyway? Even if the user can't add new profiles they could view existing ones

**Comment by Mte90** [comment]
View no because now is a db sqlite that is removed at every halt.
It is better to do a function that check from the `config.yml` if the utility exist so we can fix it also for the others.

**Comment by tomjn** [comment]
In that case, perhaps modify the vagrantfile to check if the folder is present and add the domain

**Comment by Mte90** [comment]
Well in that way the utility is available but not getting any updates or provisioning, this can generate issues.
